.github/workflows/ci.yaml

```
@@ -0,0 +1,44 @@
+name: Continuous Integration
+
+on:
+    push:
+        branches: [master]
+    pull_request:
+        branches: [master]
+
+jobs:
+    test:
+        strategy:
+            matrix:
+                os: [ubuntu-latest, macos-latest]
+
+        runs-on: ${{ matrix.os }}
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use Node.js 12
+              uses: actions/setup-node@v1
+              with:
+                  node-version: 12
+            - run: yarn install
+            - name: Test
+              uses: GabrielBB/xvfb-action@v1
+              with:
+                  run: yarn test
+
+    release:
+        needs: test
+
```

It may be better if we only do the release job when a release is created

```suggestion
        if: success() && startsWith( github.ref, 'refs/tags/releases/')
```

---

Actually, this is the semantic-release job. We enforce useful commit messages so semantic-release is able to know if a release should be created or not. Therefore, we do not have to manage versions by ourselves anymore.

---

I suggest you take a look at the `semantic-release` [docs](https://github.com/semantic-release/semantic-release), it really worths the read. :)